packages/nxengine/build.sh

```diff
@@ -0,0 +1,39 @@
+TERMUX_PKG_NAME=NXEngine
+TERMUX_PKG_HOMEPAGE=https://nxengine.sourceforge.net
+TERMUX_PKG_DESCRIPTION="Open-source rewrite engine of the Cave Story for Dingux and MotoMAGX"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@Yonle"
+TERMUX_PKG_VERSION=1.0.0.4-Rev4
+TERMUX_PKG_SKIP_SRC_EXTRACT=true
+TERMUX_PKG_DEPENDS="sdl, sdl-ttf, pulseaudio"
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_get_source() {
+	git clone https://github.com/EXL/NXEngine $TERMUX_PKG_SRCDIR
+}
+
+termux_step_make() {
+	export TERMUX_PREFIX
+        make -ef Makefile.linux
+}
+
+termux_step_make_install() {
+	curl -Lo $TMPDIR/nxengine.tar.gz https://github.com/EXL/NXEngine/releases/download/v1.0.0.4-Rev4/LIN32-NXEngine-1.0.0.4-Rev-4.tar.gz
+	cd $TERMUX_PREFIX/share
+	tar -xvzf $TMPDIR/nxengine.tar.gz
+	rm $TMPDIR/nxengine.tar.gz
```

Comment:
What is that?

Reply:
well a script that used to get Doukutsu assets that required by NXEngine, such as character looks, music, and etc.

*P.S. The build does NOT has it's game assets placed on it*